test/integration/graql/reasoner/stubs/resourceAttachment.gql

```diff
@@ -30,6 +31,7 @@ derived-resource-string sub derivable-resource-string, value string;
 resource-long sub attribute, value long;
 derived-resource-boolean sub attribute, value boolean;
 subResource sub reattachable-resource-string;
+unrelated-reattachable-string sub derivable-resource-string, value string;
```

Comment:
I suggest that this should `sub attribute` to be absolutely sure that it works outside of a hierarchy. At present the two types in the new test share a parent

Reply:
sorted, still works!